src/Neo/SmartContract/Native/CryptoLib.cs

```diff
@@ -63,6 +64,45 @@ public static byte[] Murmur32(byte[] data, uint seed)
             return murmur.ComputeHash(data);
         }
 
+
+        /// <summary>
+        /// Computes the script hash from specified public key byte array.
+        /// </summary>
+        /// <param name="pubKey">The public key to be used.</param>
+        /// <param name="curve">The curve to be used by the ECDSA algorithm.</param>
+        /// <returns>The computed public key script hash. Zero in case of error.</returns>
+        [ContractMethod(CpuFee = 1 << 15)]
+        public static UInt160 PubKeyToScriptHash(byte[] pubKey, NamedCurve curve)
```

Comment:
System.Contract.CreateStandardAccount?

Reply:
This is correct, thank you, will remove this.